apps/app/src/views/components/SplitLayout.tsx

```diff
@@ -10,14 +10,14 @@ export const SplitLayout = (props: any) => {
     <Stack
       flexDir={isMobileAndTablet ? 'column' : 'row'}
       gap={5}
-      width="95vw"
+      width="100%"
```

Comment:
this should be `%` instead of `vw` as we want the stack to fill the width of the parent, not the viewport width.